recipe/meta.yaml

```diff
@@ -59,6 +65,7 @@ test:
     - {{ compiler('c') }}  # [linux or win]
```

Comment:
pytest >=8.0 and pytest-xdist + psutil can be added.
https://github.com/sphinx-doc/sphinx/blob/v8.2.3/pyproject.toml#L110-L116